__tests__/news.test.js

```diff
@@ -132,6 +136,27 @@ describe('GET /api/articles', () => {
                 expect(article).toHaveProperty('comment_count')
         })
     });
+})  
+    test('should return correct data types', () => {
+    return request(app)
+    .get(`/api/articles`)
+    .expect(200)
+    .then((response) => {
+        const responseArray = response.body.article
+        const currentArticleLength = data.articleData.length
+        expect(responseArray).toHaveLength(currentArticleLength)
+        responseArray.forEach((article) => {
+            expect(Object.keys(article)).toHaveLength(8)
+            expect(typeof article.author).toEqual('string')
+            expect(typeof article.title).toEqual('string')
+            expect(typeof article.article_id).toEqual('number')
+            expect(typeof article.topic).toEqual('string')
+            expect(typeof article.created_at).toEqual('string')
+            expect(typeof article.votes).toEqual('number')
+            expect(typeof article.article_img_url).toEqual('string')
+            expect(typeof article.comment_count).toEqual('number')
```

Comment:
Please take a look at the jest matchers expect.objectContaining or .toMatchObject going forward.